.github/workflows/main.yml

```diff
@@ -5,10 +5,6 @@ on:
     branches:
       - '**'
   create:
-    branches:
-      - 'master'
-    tags:
-      - '**'
```

Comment:
Why was this removed? I'd assume this is useful for Bizon to tag a release and then instantly use the artifacts from it for it to upload to TestFlight or something.

Reply:
As the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore) says (and *actionlint* agrees), the `tags` key can only be inside the `push` key. I didn't add it there, because I didn't want the CI to change, I just wanted to clean it up.

Reply:
Looking in the doc again, it seems that the `on.create` key is also invalid

Reply:
This will change the CI though.